talk.md

```diff
@@ -524,6 +523,53 @@ http://choosealicense.com
 <!--many options.  Go to next slide -->
 ---
 
+
+##  Creative Commons licenses
+- CC0
+ - public domain
+ - waive all interests that may exist worldwide
+ - no Rights Reserved
+- CC
+ - Attribution (by)
+  - must give credit to author/s        
+ - ShareAlike (sa)
+  - distribute any modified work on the same terms
+ - NonCommercial (nc)
+  - no commercially use without permission
+ - NoDerivatives (nd)
+  - no modifications without permission
+
+source:https://creativecommons.org
+
+---
+### Permissive
+- MIT
+ - lets people do anything they want with your code as long as they provide attribution back to you and don’t hold you liable.
+- BSD
+ - lax (not strict), permissive, non-copyleft, free
+- CC-BY
+ - Only need to give credit to author/s
+- Apache License
+ - Like MIT but also provides an express grant of patent rights from contributors to users
+---
+
+### Share Alike
+- Creative Commons Attribution-ShareAlike (a copyleft, free content license)
+- Creative Commons Attribution-NonCommercial-ShareAlike (a proprietary license).
```

Comment:
but here CC license are mixed inbetween software licenses - the share-alike software licenses should be LGPL and MPL, not CC

Reply:
This is confusing for me. Need to read more, what was your source ?
I got this from : https://en.wikipedia.org/wiki/Share-alike

Reply:
For me the sources are given at the end of the lecture. This is what I have read. But we have to make sure that we do not confuse software licenses and CC licenses. Already the one CC slide can be confusing and one could consider removing but I thought it would be good to at least mention CC licenses somewhere but they are typically not for software.

Reply:
Moved the CC to assorted topics